.gitignore

```diff
@@ -99,3 +99,9 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Ignore package lock files generated by npm and Yarn

```

Comment:
no need to ignore those files here, just need to remove the changes from the commit

Reply:
ok